submodules/LIB/hardware/syn/yosys/build.tcl

```diff
@@ -0,0 +1,13 @@
+ 
+
+# Read the Verilog files
+
+yosys read_verilog -I../simulation/src -I../src -defer ../src/*.vh ../src/*.v
```

Comment:
this is a synthesis tool. Why doe it need to read simulation files ??

Reply:
yosys read_verilog -I../src ../src/*.v

this makes more sense

Reply:
solved